<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameUnitSettingsDialog</class>
 <widget class="QWidget" name="GameUnitSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>820</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unit settings</string>
  </property>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>780</y>
     <width>140</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>780</y>
     <width>140</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxUnitSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>370</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>Unit settings</string>
   </property>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>21</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelStrength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Strength:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStrength">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>61</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMovementPoints">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>103</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Movement points:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMovementPoints">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelOwner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>143</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Owner:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOwner">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>200</width>
      <height>28</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditStrength</tabstop>
  <tabstop>lineEditMovementPoints</tabstop>
  <tabstop>lineEditOwner</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
